### PR TITLE
[MRG] Adding more tests

### PR DESCRIPTION
**Pull request by gedankenstuecke**

### CODE REVIEW

**Comment by gedankenstuecke** [comment]
Do we care about the block-size in these cases?

**Comment by tsujigiri** [comment]
I'd say no.

**Comment by gedankenstuecke** [comment]
K, should be good to merge then 👍

File: spec/features/password_change_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    end
+
+    scenario 'finally got it right' do
+      visit "/users/#{user.id}/changepassword"
+      fill_in('Password', with: 'QueenAnnesRevenge')
+      fill_in('Confirmation', with: 'QueenAnnesRevenge')
+      click_on('Update Information')
+      expect(page).not_to have_content("Password confirmation doesn't match Password")
+      expect(page).not_to have_content('Password is too short')
+    end
```
Comment (tsujigiri) [inline spec/features/password_change_spec.rb]: For the success case, I'd add the user successfully logging in with the new password.

**Pull request merged**